api/v1alpha1/virtualmachine_types.go

```diff
@@ -299,8 +313,7 @@ type VirtualMachineStatus struct {
 	// +optional
 	Conditions []Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
 
-	// VmIp describes the IP address of the VirtualMachine.  Currently, a VirtualMachine only supports a single
-	// network interface and interface address.
+	// VmIp describes the IP address of the VirtualMachine.
```

Comment:
Possibly clarify that multiple IPs can be available for the VM, and to see the NetworkInterfaces for additional ones.

Reply:
> Possibly clarify that multiple IPs can be available for the VM, and to see the NetworkInterfaces for additional ones.

The VMODL docs that correspond is 
> Primary IP address assigned to the guest operating system, if known.

I'd make some mention of that to help users better understand what this IP is vs the others.